TheWarWithin/PriestShadow.lua

```diff
@@ -1077,6 +1077,26 @@ spec:RegisterAuras( {
         duration = 5,
         max_stack = 1
     },
+    aberrant_spellforge = {
```

Comment:
Move trinket auras to Items.lua (or Trinkets.lua, or whatever it is called) in TWW folder.